uniffi/src/interface/mod.rs

```diff
@@ -166,7 +166,7 @@ impl<'ci> ComponentInterface {
 
     pub fn ffi_string_free(&self) -> FFIFunction {
         FFIFunction {
-            name: format!("{}_string_free", self.namespace()),
+            name: format!("_{}_string_free", self.namespace()),
```

Comment:
I'm told that names with a leading underscore are reserved for use by the compiler in C code. I don't have any particular case in mind where this would cause problems for us, but one alternative might be to put the leading underscore after the namespace, like `{}__bytebuffer_free`.

Reply:
The extra character needs to be before the namespace, otherwise collisions are possible.

Assuming adding a namespace-trailing underscore, this IDL has two collisions.

```idl
namespace {
   void _string_free()
   void Sprite__object_free()
}

class Sprite {

}
```

I'll change this to something more verbose/less exotic.